### PR TITLE
update installation documentation for nu 0.60.0

### PR DESCRIPTION
**Pull request by LensPlaysGames**
### Prerequisites

- [ x] I have read and understood the `CONTRIBUTING` guide
- [x ] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x ] Docs have been added / updated (for bug fixes/features)

### Description

Nushell has been updated to `0.60.0`. This update is large and brings a large amount of breaking changes. When setting up oh-my-posh within the new-and-improved Nushell, I noticed the prompt install instructions had been deprecated by this most recent update due to the changes to the configuration system. I figured I might as well help out the next person on the same journey :^).

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

**Comment by JanDeDobbeleer** [comment]
This does imply we could add an init script for nu shell now.

File: docs/docs/install-customize.mdx [hunk 1]
```diff
@@ -104,6 +104,12 @@ config set prompt  "= `{{$(oh-my-posh prompt print primary --config ~/.jandedobb
 config set prompt  "(oh-my-posh prompt print primary --config ~/.jandedobbeleer.omp.json | str collect)"
 ```
 
+**Nu >= 0.60.0**
+
+```bash
+let-env PROMPT_COMMAND = { oh-my-posh prompt print primary --config ~/.jandedobbeleer.omp.json }
```
Comment (bdukes) [inline docs/docs/install-customize.mdx:110]:
```suggestion
let-env PROMPT_COMMAND = { oh-my-posh prompt print primary --config ~/.jandedobbeleer.omp.json }
let-env PROMPT_COMMAND_RIGHT = { oh-my-posh prompt print right --config ~/.jandedobbeleer.omp.json }
```

File: docs/docs/install-prompt.mdx [hunk 2]
```diff
@@ -148,6 +148,12 @@ config set prompt  "= `{{$(oh-my-posh prompt print primary | str collect)}}`"
 config set prompt  "(oh-my-posh prompt print primary | str collect)"
 ```
 
+**Nu >= 0.60.0**
+
+```bash
+let-env PROMPT_COMMAND = { oh-my-posh prompt print primary }
```
Comment (bdukes) [inline docs/docs/install-prompt.mdx:154]:
```suggestion
let-env PROMPT_COMMAND = { oh-my-posh prompt print primary }
let-env PROMPT_COMMAND_RIGHT = { oh-my-posh prompt print right }
```

**Review by bdukes — COMMENTED** [review]
Thanks for documenting this! I noticed that the Nu prompt included the time on the right by default, so overriding `PROMPT_COMMAND_RIGHT` helped keep everything consistent.

File: docs/docs/install-customize.mdx (hunk 1 above)
Comment (LensPlaysGames) [reply docs/docs/install-customize.mdx:110]: Thank you! I'm very new to the tool, and I appreciate how welcoming everybody is.

**Pull request closed without merging**